ssz/utils.py

```diff
@@ -15,17 +22,38 @@ def is_sedes(obj):
     return hasattr(obj, 'serialize') and hasattr(obj, 'deserialize')
 
 
+def infer_list_sedes(obj):
+    try:
+        first_element, iterator = peek(obj)
+    except StopIteration:
+        # For empty lists we can use any element sedes.
+        # Using uint32_list here as the default
```

Comment:
Since this is arbitrary, what do you think about having a special case sedes which *cannot* actually be used to prevent accidental mistakes/weirdness.  Something like `EmptyListSedes` which just always returns an empty list....